api/MediaKeyMessageEvent.json

```diff
@@ -68,7 +80,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
This seems right, it's for https://developer.mozilla.org/en-US/docs/Web/API/MediaKeySession/onmessage and that whole cluster of APIs seems to not be in IE. https://caniuse.com/eme says an older event-based model is supported, but I haven't checked what exactly that is.